utilities/JointPositionTracking/script/run_test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+echo "Welcome to the JointPositionTrackingTest. I will first move the robot and then I will store the plot the data in a file called image.png \n"
+
+echo "I'm going to move the robot. Watch out! \n"
+
+JointPositionTracking --from ../config/jointPositionTrackingOptions.ini
```

Comment:
This assumes the script to be running from a specific folder. Maybe it is worth mentioning in the README.

Reply:
You're actually right. Probably it would be nice to install also the script and the `ini`. In this case, we should be careful of the possible name collisions https://github.com/dic-iit/bipedal-locomotion-framework/pull/136#issuecomment-724300123 @traversaro